packages/x-charts/src/hooks/useTicks.ts

```diff
@@ -45,6 +45,11 @@ export interface TickParams {
    * @default 'middle'
    */
   tickLabelPlacement?: 'middle' | 'tick';
+  /**
+   * The space between ticks when using an ordinal scale. It defines the minimum distance in pixels between two ticks.
```

Comment:
Need to document how this interacts/conflicts with `tickMinStep`, `tickMaxStep`, ` tickLabelInterval` and `tickLabelMinGap`